common/persistence/visibility/visibility_manager_dual.go

```diff
@@ -177,26 +181,78 @@ func (v *VisibilityManagerDual) ListWorkflowExecutions(
 	ctx context.Context,
 	request *manager.ListWorkflowExecutionsRequestV2,
 ) (*manager.ListWorkflowExecutionsResponse, error) {
-	return v.managerSelector.readManager(request.Namespace).ListWorkflowExecutions(ctx, request)
+	if v.enableShadowReadMode() {
+		ms, err := v.managerSelector.readManagers(request.Namespace)
+		if err != nil {
+			return nil, err
+		}
+		go ms[1].ListWorkflowExecutions(ctx, request)
```

Comment:
You can silence the lint error like this:
```suggestion
		//nolint:errcheck // ignore error since it's shadow request
		go ms[1].ListWorkflowExecutions(ctx, request)
```